system-addon/content-src/components/ManualMigration/ManualMigration.jsx

```diff
@@ -0,0 +1,41 @@
+const React = require("react");
+const {connect} = require("react-redux");
+const {FormattedMessage} = require("react-intl");
+const {actionTypes: at, actionCreators: ac} = require("common/Actions.jsm");
+
+class ManualMigration extends React.Component {
+  _launchTour() {
+    this.props.dispatch(ac.SendToMain({type: at.MIGRATION_START}));
+    this.props.dispatch(ac.UserEvent({event: at.MIGRATION_START}));
+  }
+
+  _cancelTour() {
+    this.props.dispatch(ac.SendToMain({type: at.MIGRATION_CANCEL}));
+    this.props.dispatch(ac.UserEvent({event: at.MIGRATION_CANCEL}));
+  }
+
+  render() {
+    if (this.props.values.migrationExpired === true) {
+      return null;
+    }
+
+    return (<div className="manual-migration-container">
+        <p>
+          <span className="icon icon-info" />
+          <FormattedMessage id="manual_migration_explanation_msg" />
+        </p>
+        <div className="manual-migration-actions actions">
+          <button onClick={() => this._cancelTour()}>
```

Comment:
Pass in a bound-from-outside-render method to avoid unnecessary `render`. See #2928 for examples